feature-flag.md

```diff
@@ -0,0 +1,2 @@
+flag:RAW_SCREEN_SERVER
+flag:MATRIX
```

Comment:
We have to let them know that these flags exist somewhere.

Reply:
Plan B, there's also a way to add a flag [here](https://github.com/Open-Smartwatch/open-smartwatch.github.io/blob/3357dbb630603e2e33255d887dbf3b32512fa566/docs/resources/firmware.md?plain=1#L5). I think we can do it.In my opinion, this is the most ideal.

Reply:
Maybe it is used - but I think the choice of `Markdown` for this config is pretty bad. Maybe take a look at YAML / JSON instead?